### ISSUE TITLE
Power World License Interactions and selecting license type 

### ISSUE THREAD

**Issue by BlueDrgonDude64** [mzy2240/ESA#114]
Using limited licenses, can lead ESA to using a license when the add-on isn't needed. 
For example, having a PW normal & transient license, but only steady state power flow is needed for ESA calculations. When running ESA the transient license is assumed to be taken, so anyone else using the same group of licenses would be unable to run transient (assuming 1 transient and multiple base PW licenses).
Basically adding choose Add-ons (from the selection of choose add-ons when opening PW) in the classes.saw.SAW() command would be the recommendation to fix this issue.

**Comment by mzy2240** [comment]
It has nothing to do with ESA. You should register the SimAuto with the proper license.
